### ISSUE TITLE
`import/no-unused-modules` does not work with VSCode

### ISSUE THREAD

**Issue by ehmicky** [benmosher/eslint-plugin-import#1422]
The following rule:

```json
"import/no-unused-modules": [2, {"unusedExports": true}]
```

does not work well with VSCode: files that have not been opened in the editor yet are not taken into account. As a consequence exports are being reported as unused even though they are used.

For example, if I just open a new VSCode Window:

![Screenshot from 2019-07-18 12-43-27](https://user-images.githubusercontent.com/8136211/61451485-c68fb100-a959-11e9-8582-18433b1431f5.png)

The `serialize()` exported function is reported, even after saving the file.

Now if I open that other file which imports the `serialize()` function:

![Screenshot from 2019-07-18 12-44-04](https://user-images.githubusercontent.com/8136211/61451498-cbecfb80-a959-11e9-91c0-69f98091e1aa.png)

Then go back to the first file and save it, the reporting is now gone:

![Screenshot from 2019-07-18 12-44-11](https://user-images.githubusercontent.com/8136211/61451505-cf808280-a959-11e9-990e-1e9caf165573.png)

It seems like only files that are open within VSCode are considered for the `import/no-unused-modules` rule.

This behavior only seems to happen with VSCode. If I run `eslint` on the command line, everything works fine.

OS: Ubuntu `19.04`
ESLint: `6.0.1`
eslint-plugin-import: `2.18.0`	
Node: `12.6.0`

**Comment by ljharb** [comment]
Sounds like you need to file an issue with vscode then.

**Comment by ehmicky** [comment]
Done here: https://github.com/microsoft/vscode-eslint/issues/717

**Comment by ljharb** [comment]
Closing for now; happy to reopen if there ends up needing to be changes here.
